File: Java/src/main/java/entities/Rating.java
package entities;

import java.util.Objects;

public class Rating {
    private int id;
    private int userId;
    private int mechId;
    private int stars;
    private String review;

    public Rating() {
        super();
    }
    public Rating(int id, int userId, int mechId, int stars, String review) {
        super();
        this.id = id;
        this.userId = userId;
        this.mechId = mechId;
        this.stars = stars;
        this.review = review;
    }
    public int getId() {
        return id;
    }
    public void setId(int id) {
        this.id = id;
    }
    public int getUserId() {
        return userId;
    }
    public void setUserId(int userId) {
        this.userId = userId;
    }
    public int getMechId() {
        return mechId;
    }
    public void setMechId(int mechId) {
        this.mechId = mechId;
    }
    public int getStars() {
        return stars;
    }
    public void setStars(int stars) {
        this.stars = stars;
    }
    public String getReview() {
        return review;
    }
    public void setReview(String review) {
        this.review = review;
    }

    @Override
    public int hashCode() {
        return Objects.hash(getId(), getUserId(), getMechId(), getStars(), getReview());
    }

    @Override
    public String toString() {
        return "Rating{" +
                "id=" + id +
                ", userId='" + userId + '\'' +
                ", mechId=" + mechId +
                ", stars=" + stars +
                ", review=" + review +
                '}';
    }
}

File: Python/entities/Rating.py
class Rating:

    def __init__(self, r_id: int, user_id: int, mech_id: int, stars: int, review: str):
        self.r_id = r_id
        self.user_id = user_id
        self.mech_id = mech_id
        self.stars = stars
        self.review = review

    def __repr__(self):
        return str({
            'id': self.r_id,
            'user_id': self.user_id,
            "mech_id": self.mech_id,
            'stars': self.stars,
            'review': self.review
        })

    def json(self):
        return {
            'Id': self.r_id,
            'UserId': self.user_id,
            "MechId": self.mech_id,
            'Stars': self.stars,
            'Review': self.review
        }